scijava-struct/pom.xml

```diff
@@ -12,7 +12,7 @@
 	<artifactId>scijava-struct</artifactId>
 
 	<name>SciJava Struct</name>
-	<description>A lightweight framework for collecting Members</description>
```

Comment:
"Structured data` feels awfully broad to me. The `README` uses the term "typed fields" - what do you think about that?

Regardless should make sure the `pom` and the `README` are consistent

Reply:
The term "typed fields" would be OK with me. From Wikipedia: "A record is a collection of fields, possibly of different data types, typically in a fixed number and sequence."